src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -64,6 +68,26 @@ void make_target_dir(mp::SSHSession& session, const std::string& target)
     run_cmd(session, fmt::format("sudo mkdir -p \"{}\"", target));
 }
 
+// Split a path string in pieces.
+void split_path(const std::string& path, std::vector<std::string>& splitting)
+{
+    std::string dir;
+
+    for (char c : path)
+    {
+        if (('\\' == c || '/' == c) && !dir.empty())
+        {
+            splitting.push_back(dir);
+        }
+        dir.push_back('\\' == c ? '/' : c);
```

Comment:
Are we ever expecting backslashes in this path?

Reply:
To be honest, I don't know. A backslash in the path is maybe a mistake of the user?

Reply:
OK, so the code is wrong, fixing it.